src/qml/QFieldLocalDataPickerScreen.qml

```diff
@@ -47,6 +47,21 @@ Page {
         parent.finished(false);
       }
     }
+
+    onOpenMenu: {
+      selectionMenu.open();
+    }
+  }
+
+  function determineMenuWidth(menu) {
```

Comment:
I like the deduplication of code, but not sure I like the fact that it ends as a function -- what about adding a QfMenu and use that code for width, it'd allow us to use it for most of our Menu {} out there.

If you agree, let's defer that as a follow up PR, this can do for now.